static/_redirects

```diff
@@ -9,3 +9,4 @@
 /legacy/helm-operator/guides/upgrade-to-ga    /legacy/helm-operator/how-to/upgrade-to-ga    301!
 /legacy/helm-operator/guides/upgrade-to-beta  /legacy/helm-operator/how-to/upgrade-to-beta  301!
 /docs/roadmap                                 /roadmap                                      301!
+/security                                     /docs/security                                301!
```

Comment:
I'm not sure I understand this.

My thinking was that we 
1. keep importing the security doc regarding processes into `/security`
2. add a new doc under `/docs/security` which explains Flux's security architecture and how to make Flux more security - which #598 is about

Both pages should link to each other.

Reply:
Sure this sounds good, will revert a few commits